PowerEditor/src/WinControls/AnsiCharPanel/ansiCharPanel.cpp

```diff
@@ -129,7 +129,7 @@ void AnsiCharPanel::insertChar(unsigned char char2insert) const
 		bool isUnicode = ((*_ppEditView)->execute(SCI_GETCODEPAGE) == SC_CP_UTF8);
 		if (isUnicode)
 		{
-			MultiByteToWideChar(0, 0, charStr, -1, wCharStr, sizeof(wCharStr));
+			MultiByteToWideChar(0, 0, charStr, -1, wCharStr, _countof(wCharStr));

```

Comment:
```
Warning	C6386	Buffer overrun while writing to 'wCharStr':  the writable size is '20' bytes, but '40' bytes might be written.
```

(this argument specifies number of characters, not number of bytes).

Reply:
See caution note at https://msdn.microsoft.com/en-us/library/windows/desktop/dd319072(v=vs.85).aspx